models/user.go

```diff
@@ -957,6 +963,11 @@ func updateUser(e Engine, u *User) error {
 
 // UpdateUser updates user's information.
 func UpdateUser(u *User) error {
+	u.Email = strings.ToLower(u.Email)
```

Comment:
could this logic be moved to `updateUser` so that all the update user logic is kept together.

Reply:
ok, done!